temp_pg_db.py

```diff
@@ -49,31 +73,40 @@ def __init__(self, databases=None, verbosity=0):
             os.mkdir(self.pg_data_dir)
             self.pg_socket_dir = os.path.join(self.pg_temp_dir, 'socket')
             os.mkdir(self.pg_socket_dir)
-            print "Creating temp PG server...",
+            printf("Creating temp PG server...")
```

Comment:
in general i'm not sure about these prints.  they are probably handy in an interactive shell, but otherwise we shouldn't be spewing stuff to stdout in the constructor. I think I need to remove these (or at least put them in `if verbosity:` blocks).